### PR TITLE
RFC: skylark transformations

### PR DESCRIPTION
**Pull request by b5**

### CODE REVIEW

File: text/0010-skylark_tranformations.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+# Summary
+[summary]: #summary
+
+TODO
+
+# Motivation
+[motivation]: #motivation
+
+Qri ("query") is about datasets. Transformions are repeatable scripts for
```
Comment (rgardaphe) [inline text/0010-skylark_tranformations.md]: Should read: "Transformations are repeatable scripts"

File: text/0010-skylark_tranformations.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+def transform(qri):
+  body = qri.get_body()
+  count = 0
+  for entry in body:
+    count += len(entry)
+  return [{"total": count}]
+```
+
+Skylark transformations have a few rules on top of skylark itself:
+- Data functions *always* return an array or dictionary/object, representing
```
Comment (osterbit) [inline text/0010-skylark_tranformations.md]: is it correct to interpret the `transform()` function of a skylark script as the "main" function (where I may also have some other function like `get_most_recent_weekday(date_list)` that returns an arbitrary type)?

Comment (b5) [reply text/0010-skylark_tranformations.md]: yep! That's exactly what's going on. I'll add a note spelling out as much

**Comment by b5** [comment]
closing. Stale RFC

**Pull request closed without merging**